.vale/styles/RedHat/RepeatedWords.yml

```diff
@@ -3,4 +3,4 @@ message: "'%s' is repeated!"
 level: error
 alpha: true
 tokens:
-  - '[^\s]+'
+  - '(\b\S+\b)\s+\b\1\b'
```

Comment:
Vale uses golang flavour of regex which doesn't support lookbacks (the "\1" part). Try this: `[^\s\.]+`
It seems to work, although I can't fully understand the regex tbh

Reply:
That works perfectly with our Vale configuration, thanks!

Reply:
> Try this: `[^\s\.]+`

Thank you, @rohennes !